octoprint_octorelay/__init__.py

```diff
@@ -349,6 +346,16 @@ def turn_off_pin(self, relay_pin, inverted, cmdOFF):
         self._logger.info("pin: {} turned off".format(relay_pin))
         self.update_ui()
 
+    def turn_on_pin(self, relay_pin, inverted, cmdON):
+        GPIO.setup(relay_pin, GPIO.OUT)
```

Comment:
@NDR008 , is it ok, that exactly the same command is being executed right before calling this function, in line 318?

https://github.com/borisbu/OctoRelay/blob/cd52345b03b61de12cfd26eb275bf42d0441f47c/octoprint_octorelay/__init__.py#L318

Reply:
made a suggestion to keep only this one